map/style_tests/style_symbols_consistency_test.cpp

```diff
@@ -96,3 +124,60 @@ UNIT_TEST(Test_SymbolsConsistency)
 
   TEST(res, ());
 }
+
+UNIT_TEST(Test_ClassSymbolsConsistency)
+{
+  // Tests that symbols specified for object class are same for all zoom levels,
+  // e.g. lawyer object class has 'lawyer' icon for all zoom levels.
+  // Also test checks that visible scales are consecutives - without holes:
+  // e.g. 3,4,5 - ok, but 4,5,7 - not ok (6 is missing).
+
+  bool res = true;
+  styles::RunForEveryMapStyle([&res](MapStyle mapStyle) {
+    for (auto const & classifMapping : GetCurrentStyleSymbolsMapping())
+    {
+      auto const & scaleAndSymbolMapping = classifMapping.second;
+
+      bool invalidSymbol = false;
+      bool invalidScale = false;
+
+      pair<int, string> prev = *scaleAndSymbolMapping.begin();
+      for (auto itr = ++scaleAndSymbolMapping.begin(); itr != scaleAndSymbolMapping.end(); ++itr)
```

Comment:
auto - is the 'pair<const int, string>' - not reassignable, therefore it is 'pair<int, string>'